CONTRIBUTING.md

```diff
@@ -65,14 +65,59 @@ with `fix:`, `feat:` or similar according to
 
 ## <a name="releasing">Releasing (maintainers only)</a>
 
+### Short version (I've done this before!)
+1. Visit the [GitHub Actions tab](https://github.com/muxinc/media-chrome/actions) and 
+select the "CD" action in the left sidebar.
+2. Click the "Run workflow" dropdown and choose the correct `Version` on the `main` branch.
+   - In the **Use workflow from** select box, make sure **Branch: main** is selected.
+   - In the **Version** select box, choose the appropriate value:
+      - If the commit messages in this release were written using the correct conventional commit style, select `conventional`.  
+      - If the commit messages aren't accurate, manually choose the correct semver version `patch`, `minor`, `major`.
+3. When you're confident with your choices, click the green **Run workflow** button to start the release process.
+4. After a few minutes, a new release will be published. This includes an NPM package, new version tags, and a GitHub release.
+
+---
+### Long version (I need more context!)
+
 This repo uses [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
-and Github Actions for continuous deployment (CD).
+and GitHub Actions for continuous deployment (CD).
+
+> If you're unfamiliar with conventional commits, it's a good idea to review the link above before continuing.
+
+Here's a quick summary of how we use conventional commits in this repository:
+
+- Commit messages prefixed with `fix:` will notify CD that the release is minimally a `patch` release.
+- Commit messages prefixed with `feat:` will notify CD that the release is minimally a `minor` release.
+- Major releases must always be designated manually during the Release process.
```

Comment:
I believe "BREAKING CHANGE" in the footer of commits will designate a major release.

Reply:
@cjpillsbury can you comment on this?

Reply:
Ah @gkatsev is right, we just haven't relied on this feature yet afaik (See: https://www.conventionalcommits.org/en/v1.0.0/#summary)

Reply:
the shorthand is including a `!` after the type. e.g.:
`feat(css-vars)!: Refactor our CSS custom properties (aka "CSS vars") for consistency.`

Reply:
We'd need to confirm whether lerna supports that or not.

Reply:
I'm going to assume we can't use it since the default parser doesn't seem to include it in here https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-angular/parser-opts.js#L4

Reply:
booooo. We should issue a pull request at some point.